tests/e2e/vsphere_volume_expansion.go

```diff
@@ -1619,11 +1616,11 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 
 		ginkgo.By("File system resize should not succeed since SPS service is down. " +
 			"Expecting an error")
-		expectedErrMsg := "VolumeResizeFailed"
-		isFailureFound, err := waitForEventWithReason(client, namespace, pvclaim.Name, expectedErrMsg)
+		expectedMsg := "FileSystemResizeRequired"
```

Comment:
I am not sure if this testcase is valid anymore. If the expected message is coming FileSystemResizeRequired that means volume expansion has succeeded instead of an error.